NEWS.md

```diff
@@ -175,6 +175,8 @@ Deprecated or removed
   * The forms of `read`, `readstring`, and `eachline` that accepted both a `Cmd` object and an
     input stream are deprecated. Use e.g. `read(pipeline(stdin, cmd))` instead ([#22762]).
 
+  * The default `juliarc.jl` file on Windows has been removed. Use `joinpath(JULIA_HOME, "7z.exe")`
+    if you need access, e.g., to `7z.exe` on Windows ([#21540]).
```

Comment:
there's nothing specific to 7z about this - it's the case for any executable or library in julia's bin dir